java-frontend/src/test/files/se/ConditionAlwaysTrueOrFalseCheck.java

```diff
@@ -1613,13 +1613,12 @@ void testAfterAddAssignment(int y) {
   }
 }
 
-public class TryCatchCFG {
+abstract class TryCatchCFG {
 
   private Object monitor;
   private boolean shutdown;
 
-  private void doSomething() {
-  }
+  abstract void doSomething();
```

Comment:
I am really wondering about this modification : If I'm correct that means a method invocation like this one might not throw an exception/error whereas it could/should be the case. Maybe this is the correct behavior but might worth to have second thoughts (and then a test ?) about that.